frontend/src/metabase/setup/tests/premium.unit.spec.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint jest/expect-expect: ["error", { "assertFunctionNames": ["expect", "expectSectionToHaveLabel", "expectSectionsToHaveLabelsInOrder"] }] */
+
+import { createMockTokenFeatures } from "metabase-types/api/mocks";
+
+import type { SetupOpts } from "./setup";
+import {
+  expectSectionsToHaveLabelsInOrder,
+  expectSectionToHaveLabel,
+  setup,
+  skipWelcomeScreen,
+} from "./setup";
+
+const setupPremium = (opts?: SetupOpts) => {
+  return setup({
+    ...opts,
+    hasEnterprisePlugins: true,
+    tokenFeatures: createMockTokenFeatures({ hosting: true }),
+  });
+};
+
+describe("setup (EE, hosting feature)", () => {
```

Comment:
when there's a valid token, the behaviour should be same as OSS, so this only checks that the license step is not shown